autoload/neomake/makers/ft/nix.vim

```diff
@@ -7,7 +7,7 @@ endfunction
 function! neomake#makers#ft#nix#nix_instantiate() abort
     return {
         \ 'exe': 'nix-instantiate',
-        \ 'args': ['%:p', '--parse-only'],
+        \ 'args': ['--parse-only'],
```

Comment:
@gilligan 
From the man page / online documentation it looked like the position of the file arg is not relevant, and therefore I've removed it - append_file will append it (and did so already before this change, i.e. the filename was there twice).